src/index.css

```diff
@@ -76,5 +76,100 @@ form {
 
 .bookstore,
 .categories {
-  margin-top: 9%;
+  margin-top: 2.52%;
+}
+
+.card {
+  background-color: white;
+  width: 85%;
+  height: 125px;
+  margin-bottom: 5px;
+  border: solid 1px gray;
+  font-size: 15px;
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+}
+
+.cards {
+  margin-left: 1%;
+}
+
+.category {
+  width: 200px;
+}
+
+.actions {
+  display: flex;
+  margin-left: 1%;
+  color: blue;
+}
+
+.comments,
+.remove {
+  margin-right: 1%;
+}
+
+.comments {
+  padding-bottom: 1%;
+}
+
+.circle {
+  background: #fff;
+  height: 80px;
+  width: 80px;
+  border: 12px solid;
+  border-radius: 50%;
+  border-color: blue blue blue gray;
+  transform: rotate(45deg);
+}
+
+.completed {
+  font-size: 25px;
+}
+
+button {
+  align-items: normal;
+  background-color: rgba(0, 0, 0, 0);
+  border-color: rgb(0, 0, 238);
+  border-style: none;
+  box-sizing: content-box;
+  color: rgb(0, 0, 238);
+  cursor: pointer;
+  display: inline;
+  font: inherit;
+  height: auto;
+  padding: 0;
+  perspective-origin: 0 0;
+  text-align: start;
+  transform-origin: 0 0;
+  width: auto;
+  -moz-appearance: none;
+  text-decoration: none;
+}
+
+.title {
+  width: 200px;
+}
+
+.category_field {
+  margin-left: 1%;
+}
+
+.cell_two {
+  display: flex;
+}
+
+.dataTables_length {
+  margin-left: 40px;
+  margin-top: 20px;
+}
+
+.dataTables_filter {
+  margin-right: 60px;
+  margin-top: 20px;
+}
+
+.dataTables_paginate {
+  margin-right: 60px;
 }
```

Comment:
- [ ] Styling should be EXACTLY the same as the one provided on [Zeplin]()

your page
![ms](https://user-images.githubusercontent.com/44745855/132500619-a3ce3b9d-fbe2-4635-a3b2-4a18bc128547.png)

zeplin
![sd](https://user-images.githubusercontent.com/44745855/132501012-63ccab0a-7249-46d7-b359-cc0f1f0e6cf9.png)

form
![form](https://user-images.githubusercontent.com/44745855/132501090-6480d1a8-52d0-4667-8d38-0081d5306266.png)

Reply:
- [ ] Please try to make it looks exactly like the design like colors, font, layouts always compare your page
with the design

Reply:
**Changes Required**
- [ ] Nice try getting closer to the Zeplin design, However you can still do better with the typography, layout and color in regards to making it look as close as possible to the Zeplin design. Let us try to make these changes. Below is how your page looks like,I think you can reduce on the color contrast.  In case you're  struggling with getting the right color, I suggest you install a color picker extension and use it to pick the colors from the Zeplin design.

![your-page](https://user-images.githubusercontent.com/7220167/132524330-9293f3d1-b0cb-448b-979c-f938ade6c7a7.png)